### DISCUSSION TITLE
Input Fields - Label prop for A11y

### DISCUSSION THREAD

**Discussion by megaroeny** [primefaces/community#70] [Figma UI Kit]
In the Figma kit, you can only show a label during `focus` with a `Floating label`. Can you provide a boolean prop always to have a label visible? This is also a requirement for AA accessibility criteria. It would also provide more flexibility with the input field props overall (in Figma). 

Something helpful you can do is to create a "label" atomic-level component (hidden in the library with an _), so that you can insert that in every variant. Makes it easier to maintain and also expose as a prop.

---

I also noticed in the code components you have an [example for "Help Text"](https://primeng.org/inputtext#helptext) that shows a persistent label being displayed, same as my request above. This is basically what I am requesting:
![Screenshot 2023-09-19 at 9 32 51 AM](https://github.com/primefaces/community/assets/1559447/496e00d9-e0be-4751-85a1-361fa76308c8)

Thanks!

**Comment by cagataycivici** [comment]
I've pinged the design team to join for assistance.

**Comment by onursenture** [comment]
We're currently working on the tokens update for the UI Kit but noted for the next releases. Thank you.

**Reply by megaroeny** [reply]
No worries at all. Totally aware. Just want to add something to your backlog as I catch things. Thanks!
